monads.gemspec

```diff
@@ -15,5 +15,6 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ['lib']
 
+  spec.add_runtime_dependency 'bundler', '~> 1.7'
```

Comment:
I don’t understand why this is here, and don’t think it should be. Bundler is not a runtime dependency of this gem.